CMakeLists.txt

```diff
@@ -13,13 +13,44 @@ endif()
 
 project(miopengemm)
 
-find_package(ROCM REQUIRED PATHS /opt/rocm)
+#find_package(ROCM REQUIRED PATHS /opt/rocm)
```

Comment:
@JehandadKhan @arvindcheru 
Why would it be necessary to download rocm-cmake here?

Reply:
Response from the reviewee:

> Currently the backward support is enabled using this rocm-cmake function, since this is also used in MIOpenGemm, added this code to download if the required version is not found
> To my knowledge this can be avoided, once we have rocm-cmake 0.7.3 or above available in all build env